angular/src/app/app.module.ts

```diff
@@ -82,17 +77,20 @@ import '../styles/headings.css';
      *
      * See: https://github.com/ngrx/effects/blob/master/docs/api.md#run
      */
-    //EffectsModule.run(BookEffects),
-    //EffectsModule.run(CollectionEffects),
+    // EffectsModule.run(BookEffects),
+    // EffectsModule.run(CollectionEffects),
 
     /**
      * `provideDB` sets up @ngrx/db with the provided schema and makes the Database
      * service available.
      */
-    DBModule.provideDB(schema)
+    DBModule.provideDB(schema),
```

Comment:
Misschien is dit een goed moment om dit soort overbodige modules er uit te halen.